during_workshop.md

```diff
@@ -212,10 +183,14 @@ The function now expects us to send a JSON object to it containing the subtitle
 
 ![Thunder Client](./images/Thunder-Client-SendRequest.png) 
 
-Take a moment to customise the "httpTrigger" binding a little.
-- We don't want to accept GET requests, so remove that from the list of accepted methods in `function.json`.
-- Rename the binding from "req" to "request". This means updating the "name" property for that binding in `function.json` as well as the corresponding function parameter in `__init__.py`. Each binding in `function.json` needs a corresponding parameter in the `main` function in `__init__.py`, except one output binding can use `main`'s return value instead, if it has name equal to "$return".
+Take a moment to customise the "HttpEndpoint" function a little (by adding parameters to `@app.route`).
+- We only want the function to accept POST requests, so specify this as a `methods` parameter in `@app.route`
+- Rename the argument of the function from `req` to `request` (just changing the name of the argument in the function will **not** work!)
  
+> For the best experience we recommend setting up code completion by opening a terminal in the AcmeSubProject folder and install the dependencies by running `pip install -r requirements.txt`. You should then be able to test this is working by typing `@app.` and seeing the autocompletion options:
+> ![Autocomplete example](images/Azure%20Functions%20Autocomplete.png)
+> If this isn't working it's worth trying to restart the Python interpreter ("Python: Restart Interpreter" from the VSCode Command Window)

```

Comment:
But then this did fix an import complaint that was showing up.. so maybe you were right 🤔